### DISCUSSION TITLE
The page keeps refreshing

### DISCUSSION THREAD

**Discussion by MHuiG** [community/community#3081] [Discussions] [answered]
demo: https://github.com/volantis-x/hexo-theme-volantis/discussions/549

This is a discussion converted from issues.

win10/chrome

After the page opened, it began to refresh continuously

**Comment by MHuiG** [comment, marked as answer]
I locked the conversation in this discussion, then unlocked it, and it's back to normal now.

**Reply by evi-liu** [reply]
Glad it's working now!
